README.md

```diff
@@ -10,9 +10,11 @@ It is mainly based on `AllenNLP` and `transformers`.
 ## Installation
 The following command installs all necessary packages:
 ```.bash
-pip install -r requirements.txt
+conda create --name <Environment_name> python=3.8
+conda activate <Environment_name> 
+python install -e .
```

Comment:
```suggestion
pip install -e .
```

Does this work? Do you not need the `--no-dependencies` flag?

Reply:
Dang, I didn't even see this.

Reply:
It works for me without the flag.

Reply:
I think we only want (not need) the flag if the dependencies were installed with Conda before installing this package. The issue is we don't want `pip` upgrading/downgrading things willy-nilly.

Reply:
uhh.. I tested with pip. Will modify it.